litellm/utils.py

```diff
@@ -5143,7 +5146,7 @@ def get_supported_openai_params(model: str, custom_llm_provider: str):
     """
     if custom_llm_provider == "bedrock":
         if model.startswith("anthropic.claude-3"):
-            return litellm.AmazonAnthropicClaude3Config().get_supported_openai_params()
+            return litellm.AmazonAnthropicClaude3Config().get_supported_openai_params() + ["extra_headers"]
```

Comment:
@themrzmaster can you remove this line. 

It'll send the extra headers as part of the request body during calls

Reply:
you've already solved your problem by taking it in as a direct param into the completion() function in main.py

Reply:
if i remove, i get 
` bedrock does not support parameters: {'extra_headers'}`
@krrishdholakia

Reply:
any other idea how to avoid this error?

Reply:
@krrishdholakia it would really help me if we could ship that soon. Is there anything I can do?

Reply:
i changed it to bedrock.py